src/Time/Units.hs

```diff
@@ -100,6 +114,41 @@ instance KnownSymbol (ShowUnit unit) => Show (Time unit) where
                                                 n -> '/' : show n
                       in numeratorStr ++ denominatorStr ++ symbolVal (Proxy @(ShowUnit unit))
 
+----------------------------------------------------------------------------
+-- Creation helpers
+----------------------------------------------------------------------------
+
+sec :: Natural -> Second
```

Comment:
It would be good to add documentation with single example for every function. Something like this one:

```haskell
-- | Creates 'Second' from given 'Natural'.
-- >>> sec 42
-- 42s :: Second
```

And then just copy-paste and change interesting parts.